server/src/main/java/org/apache/iotdb/db/conf/adapter/ActiveTimeSeriesCounter.java

```diff
@@ -59,11 +59,12 @@ public void init(String storageGroup) {
 
   @Override
   public void offer(String storageGroup, String device, String measurement) {
+    String path = device + "." + measurement;
```

Comment:
It's better to replace "." with `IoTDBConstant.PATH_SEPARATOR`.

Reply:
Fixed